contractdata/PARIS/oldparischal.json-bak

```diff
@@ -0,0 +1,5449 @@
+{
```

Comment:
What is this file?

Reply:
Thanks for spotting this. This is a backup file of the original `_PARIS_CHALLENGES.json` that I forgot to delete.